### PR TITLE
osa - squash images

### PR DESCRIPTION
**Pull request by mjudeikis**
Squash azure images to single image

### CODE REVIEW

File: ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml [hunk 1]
```diff
@@ -13,70 +13,14 @@ build_root:
     tag: latest
 canonical_go_repository: github.com/openshift/openshift-azure
 images:
-- dockerfile_path: images/sync/Dockerfile
+- dockerfile_path: images/azure/Dockerfile
```
Comment (charlesakalugwu) [inline ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml]:
```suggestion
- dockerfile_path: images/arho/Dockerfile
```

File: ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml [hunk 2]
```diff
@@ -13,70 +13,14 @@ build_root:
...
-- dockerfile_path: images/canary/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/src/github.com/openshift/openshift-azure/canary
-  to: canary
+        source_path: /go/src/github.com/openshift/openshift-azure/azure
+  to: azure
```
Comment (charlesakalugwu) [inline ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml]:
```suggestion
  to: arho
```

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml [hunk 3]
```diff
@@ -14,13 +14,7 @@ postsubmits:
...
         - --give-pr-author-access-to-namespace=true
         - --secret-dir=/usr/local/e2e-secrets/azure
-        - --target=[output:stable:azure-controllers]
-        - --target=[output:stable:canary]
-        - --target=[output:stable:etcdbackup]
-        - --target=[output:stable:metricsbridge]
-        - --target=[output:stable:startup]
-        - --target=[output:stable:sync]
-        - --target=[output:stable:tlsproxy]
+        - --target=[output:stable:azure]
```
Comment (charlesakalugwu) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml]:
```suggestion
        - --target=[output:stable:arho]
```

Comment (charlesakalugwu) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml]: and the rest below...

File: ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml (hunk 1 above)
Comment (kwoodson) [reply ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml]: I'm unfamiliar with the namespace arho.  What/why this change?

Comment (jim-minter) [reply ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml]: @kwoodson this combines with https://github.com/openshift/openshift-azure/pull/1464 to reduce the number of images we ship to simplify the release train

**Comment by jim-minter** [comment]
/lgtm

taking the plunge...

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml [hunk 4]
```diff
@@ -511,13 +463,7 @@ presubmits:
...
-        - --target=[output:stable:azure-controllers]
-        - --target=[output:stable:canary]
-        - --target=[output:stable:etcdbackup]
-        - --target=[output:stable:metricsbridge]
-        - --target=[output:stable:startup]
-        - --target=[output:stable:sync]
-        - --target=[output:stable:tlsproxy]
+        - --target=[output:stable:azure]
         - --target=cluster-launch-e2e-azure-reimagevm
         - --template=/usr/local/cluster-launch-e2e-azure-reimagevm.yaml
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml]: Why do you need to literally target the others? CI Operator is supposed to be running the entire dependency tree for you and there is a mechanism to opt into waiting on images for templates automatically.

**Pull request merged**